bin/BuildPackages.sh

```diff
@@ -85,6 +98,7 @@ make TOPDIR=`pwd`
 chmod -R a+rX .
 cd bin
 aa=`./config.guess`
+# TODO dangerous to parse output of ls, use find instead
```

Comment:
But I still don't quite understand what the `ln -s` below is meant to do... What is it needed for? Perhaps there is a better solution...